modules/serverless-deleting-broker-injection.adoc

```diff
@@ -1,12 +1,12 @@
 // Module included in the following assemblies:
 //
-// * serverless/knative_eventing/serverless-using-brokers.adoc
+// * /serverless/develop/serverless-using-brokers.adoc
 
 :_content-type: PROCEDURE
 [id="serverless-deleting-broker-injection_{context}"]
 = Deleting a broker that was created by injection
 
-Brokers created by injection, by using a namespace label or trigger annotation, are not deleted permanently if a developer removes the label or annotation. You must manually delete these brokers.
+Brokers created by injection, by using a namespace label or trigger annotation, are not deleted permanently if you remove the label or annotation. You must manually delete these brokers.
```

Comment:
This first sentence is a bit hard to follow. Perhaps consider using active voice? You might also consider breaking it up into two sentences. I gave it a go but couldn't come up with something worth sharing. 😁

Reply:
Had a go at rewording this, lmk what you think 🙂